pkg/isoeditor/isoutil.go

```diff
@@ -146,6 +146,24 @@ func Create(outPath string, workDir string, volumeLabel string) error {
 				},
 			},
 		}
+	} else if exists, _ := fileExists("images/efiboot.img"); exists {
+		// Creating an ISO with EFI boot only
+		efiSectors, err := efiLoadSectors(workDir)
+		if err != nil {
+			return err
+		}
+		options.ElTorito = &iso9660.ElTorito{
+			BootCatalog:     "boot.catalog",
```

Comment:
Should we check that this file exists? It doesn't in the x86 isos.

Reply:
I know it won't affect them. My point is just that it doesn't exist in some cases so we should check for it.

Reply:
Yeah, so I've meant that it should always exist if the iso has only one boot file afaik. But can add a check just in case.

Reply:
Added.